atomic_reactor/plugins/pre_pin_operator_digest.py

```diff
@@ -132,6 +141,20 @@ def run_in_orchestrator(self):
         else:
             if pullspecs:
                 replacement_pullspecs = self._get_replacement_pullspecs(pullspecs)
+
+                if self.operator_csv_modifications_url is not None:
```

Comment:
@tkdchen;
I will end up needing similar code (ending with `modification_data = resp.json()`) to modify other metadata. May I ask you to make this reusable in some way?